src/Estimators/InputSection.h

```diff
@@ -137,6 +152,19 @@ class InputSection
   }
 
 protected:
+  /** reads attributes for both the root node and parameter/child nodes
+   *  that aren't delegated.
+   *
+   *  Side effect only method updates values_.
+   *  \param[in]   cur              current xml node
+   *  \param[in]   consume_name     parse the name attribute (this has complicated semantics in QMCPACK input)
+   *                                when a parameter has an ename="parameter" and the name attribtue is used to identify the
```

Comment:
Should this be 'name="parameter"'?  I'm not sure if this is referring to the input file (in which case it would be 'name') or the code (which uses 'ename', which I assume stands for  "estimator name"?)

Minor: spelling of 'attribute'

Reply:
ename is element name so the case here is:
```
<parameter name="center"> 0.0 0.0 0.0 </parameter>
```
Although we also allow
```
<center> 0.0 0.0 0.0 </center>
```
In this case ename="center"

The semantics of "name" elsewhere in input are as the identifier for a input element that has multiple instances.
Since the name of the parameter is actually equivalent to ename for all the other tags we don't consume it when its the name a parameter because its already getting used as the key or qualifier extension for the parameter value(s).